### PR TITLE
adds npm_package/lib to node_path

### PR DESCRIPTION
**Pull request by kyle-hall**
I had to add this to the NODE_PATH to finish the process. Before I did, I got an error telling me my NODE_PATH and npm root were different and to make this change to fix that. It was the global installation of yo that gave this error message.

### CODE REVIEW

File: npm-global-without-sudo.md [hunk 1]
```diff
@@ -20,12 +20,15 @@ prefix=${HOME}/.npm-packages
 
 ```sh
 NPM_PACKAGES="${HOME}/.npm-packages"
+#add NPM_PACKAGES to NODE_PATH
+export NODE_PATH="NODE_PATH:$NPM_PACKAGES/lib/node_modules"
```
Comment (sindresorhus) [inline npm-global-without-sudo.md]: You're missing a `$`

File: npm-global-without-sudo.md [hunk 2]
```diff
@@ -20,12 +20,15 @@ prefix=${HOME}/.npm-packages
 
 ```sh
 NPM_PACKAGES="${HOME}/.npm-packages"
+#add NPM_PACKAGES to NODE_PATH
```
Comment (sindresorhus) [inline npm-global-without-sudo.md]: space after `#`

File: npm-global-without-sudo.md [hunk 3]
```diff
@@ -20,12 +20,15 @@ prefix=${HOME}/.npm-packages
...
+export NODE_PATH="NODE_PATH:$NPM_PACKAGES/lib/node_modules"
 
 PATH="$NPM_PACKAGES/bin:$PATH"
 
 # Unset manpath so we can inherit from /etc/manpath via the `manpath` command
 unset MANPATH # delete if you already modified MANPATH elsewhere in your config
 export MANPATH="$NPM_PACKAGES/share/man:$(manpath)"
+
```
Comment (sindresorhus) [inline npm-global-without-sudo.md]: ?

Comment (kyle-hall) [inline npm-global-without-sudo.md]: Yeah, I'm not sure what happened for that line.

**Comment by kyle-hall** [comment]
@sindresorhus I'll get those comments fixed up. Thanks for looking at the PR

File: npm-global-without-sudo.md (hunk 1 above)
Comment (kyle-hall) [inline npm-global-without-sudo.md]: Oddly enough, it does work without '$." I had it exactly this way in my .bashrc and it worked okay. It does make more sense to have it there, though, and I tested the change to make sure it works. So I'm making the change as suggested.

**Comment by kyle-hall** [comment]
@sindresorhus Suggested fixes have been made. Looks good in the preview.

**Comment by sindresorhus** [comment]
Actually, I'm not about this. Setting NODE_PATH is a pretty bad practice.

@SBoudrias Why do we require this again?

**Comment by kyle-hall** [comment]
@sindresorhus Could you explain why setting NODE_PATH is a bad practice?

**Comment by sindresorhus** [comment]
https://www.google.com/search?q=NODE_PATH+bad

**Comment by SBoudrias** [comment]
> Why do we require this again?

We don't require it.

Yeoman supports it because some people rely on it. That being said, misconfigured NODE_PATH is the main source of issues with people not finding generators on their computer.

**Comment by kyle-hall** [comment]
@SBoudrias Okay, I was basing this on the error message from the global yo install. I thought the error meant it was required. Thanks for clarifying.

**Pull request closed without merging**